dlp/snippets/inspect/inspect_test.go

```diff
@@ -681,3 +688,207 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
 		t.Errorf("TestInspectTableWithCustomHotword got %q, want %q", got, want)
 	}
 }
+
+func TestInspectDataStoreSendToScc(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+	u := uuid.New().String()[:8]
+	datastoreNamespace := "golang-samples" + u
```

Comment:
nit: use `fmt.Sprintf()` instead of string concatenation.

Reply:
Updated it.